GoogleForCreators/web-stories-wp
# Animations: Add Pulse animation effect and part

## Summary

This PR adds the Pulse Effect _and_ Pulse Animation Part to our repo.

## Relevant Technical Choices

I realized when looking at the AMP Pulse animation that we'd need a new animation part in order to pull it off, so I created a new animation part and then wrapped the effect around it.

## User-facing changes

None

## Testing Instructions

1.) Go to Story Book "Animations -> Effects -> Pulse".
2.) Toggle between both pages and notice the pulsing glory (one from AMP and the other from us).

**NOTE**:  I couldn't figure out how to increase the amount of pulse's (or pulse scale) from the AMP side, so only our side has multiple pulses with increasing intensity going on 🎉  

Fixes #3422 

## Screenshots

![pulse_effect](https://user-images.githubusercontent.com/40646372/89075513-3be6dd80-d333-11ea-8052-831d5c8c322f.gif)
